arangod/IResearch/IResearchLink.cpp

```diff
@@ -386,75 +396,60 @@ void CommitTask::operator()() {
       THROW_ARANGO_EXCEPTION(TRI_ERROR_DEBUG);
     }
 
-    TRI_ASSERT(linkPtr->_dataStore); // must be valid if _asyncSelf->get() is valid
-    READ_LOCKER(lock, linkPtr->_dataStore._mutex); // '_meta' can be asynchronously modified
+    TRI_ASSERT(linkPtr->_dataStore);  // must be valid if _asyncSelf->get() is valid
+    READ_LOCKER(lock, linkPtr->_dataStore._mutex);  // '_meta' can be asynchronously modified
     auto& meta = linkPtr->_dataStore._meta;
 
-    commitIntervalMsec = std::chrono::milliseconds(meta._commitIntervalMsec);
-    consolidationIntervalMsec = std::chrono::milliseconds(meta._consolidationIntervalMsec);
+    commitIntervalMs = std::chrono::milliseconds(meta._commitIntervalMs);
+    consolidationIntervalMs = std::chrono::milliseconds(meta._consolidationIntervalMs);
     cleanupIntervalStep = meta._cleanupIntervalStep;
   }
 
-  if (std::chrono::milliseconds::zero() == commitIntervalMsec) {
+  if (std::chrono::milliseconds::zero() == commitIntervalMs) {
     reschedule.cancel();
 
     LOG_TOPIC("eba4a", DEBUG, iresearch::TOPIC)
-        << "sync is disabled for the link '" << id
-        << "', runId '" << size_t(&runId) << "'";
+        << "sync is disabled for the link '" << id << "', runId '"
+        << size_t(&runId) << "'";
     return;
   }
 
   TRI_IF_FAILURE("IResearchCommitTask::commitUnsafe") {
     THROW_ARANGO_EXCEPTION(TRI_ERROR_DEBUG);
   }
 
-  auto const syncStart = std::chrono::steady_clock::now(); // FIXME: add sync durations to metrics
-  auto res = linkPtr->commitUnsafe(false, &code); // run commit ('_asyncSelf' locked by async task)
+  auto res = linkPtr->commitUnsafe(false, &code);  // run commit ('_asyncSelf' locked by async task)
 
   if (res.ok()) {
     LOG_TOPIC("7e323", TRACE, iresearch::TOPIC)
-        << "successful sync of arangosearch link '" << id
-        << "', run id '" << size_t(&runId) << "', took: " << 
-        std::chrono::duration_cast<std::chrono::milliseconds>(
-          std::chrono::steady_clock::now() - syncStart).count()
-        << "ms";
-    if (code == IResearchLink::CommitResult::DONE) {
-      if (cleanupIntervalStep && cleanupIntervalCount++ > cleanupIntervalStep) { // if enabled
-        cleanupIntervalCount = 0;
+        << "successful sync of arangosearch link '" << id << "', run id '"
```

Comment:
having information about how long a particular commit has taken is useful for debugging purposes

Reply:
https://github.com/arangodb/arangodb/pull/15075/files#diff-d42e2c554656ec143789da7654d09c1ad78b9e0dc2d4f2f656e856388eb8eba3L411
You have todo about metrics I thought if we have metrics with this info we don't need this info in the log

Reply:
@MBkkt metrics endpoint can be completely disabled. Therefore a compensational log entry must present.

Reply:
Reverted these changes